src/staging/src/components/park/campingDetails.js

```diff
@@ -33,6 +36,105 @@ export default function CampingDetails({ data }) {
     setReservationsExpanded(!reservationsExpanded)
   }
 
+  // -------- Capacity Counts ----------
```

Comment:
This is duplicated from parkDates.js. Please find a way to share it between the two files.

Reply:
Thank you